src/main/res/values/strings.xml

```diff
@@ -1024,4 +1024,6 @@
 
 	<!-- 2017-11-01 -->
 	<string name="disable_replies_to_infobox_failed">Disabling replies to inbox failed</string>
+    <!-- 2017-11-19 -->
+    <string name="lang_nb_NO" translatable="false">Norsk bokmål</string>
```

Comment:
Is this string used anywhere?

Reply:
I did translate RedReader into Norwegian Bokmål